core/os/os.cpp

```diff
@@ -457,18 +457,22 @@ PackedStringArray OS::get_connected_midi_inputs() {
 	}
 
 	PackedStringArray list;
-	return list;
+	ERR_FAIL_V_MSG(list, vformat("MIDI input isn't supported on %s.", OS::get_singleton()->get_name()));
```

Comment:
I think you can remove `list` and return `PackedStringArray()` directly in the `ERR_FAIL_V_MSG`.

Reply:
Well it doesn't matter so much.